assets/xml/scenes/overworld/spot01_pal_n64.xml

```diff
@@ -2,6 +2,11 @@
     <File Name="spot01_scene" Segment="2">
         <Path Name="spot01_scenePathList_0003D0" Offset="0x03D0" NumPaths="3"/>
         <Cutscene Name="gKakarikoVillageIntroCs" Offset="0xA560"/>
+        <Cutscene Name="gKakarikoVillageRavagedCs" Offset="0x4A50"/>
+        <Cutscene Name="gKakarikoVillageNocturneOfShadowCs" Offset="0x6DF4"/>
```

Comment:
First: I would be fine with just Nocturne
Second: I would personally be fine with NocturnePart1 and NocturnePart2, since to me they are both just two parts of the same nocturne cutscene. But that opinion may not be popular

Reply:
This is something I'd like to have, but also be applied to longform cutscenes that span multiple scenes like the world creation cutscene, the light arrow cutscene, and the credits.